### ISSUE TITLE
add rerank model success for LocalAI, but no show in system

### ISSUE THREAD

**Issue by edisonzf2020** [langgenius/dify#4377]
### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] Please do not modify this template :) and fill in all the required fields.

### Dify version

0.6.8

### Cloud or Self Hosted

Self Hosted (Docker)

### Steps to reproduce

1. add rerank model mxbai-rerank-large-v1 success.
2.  can not display in system .

**Comment by gakugaku** [comment]
After configuring LocalAI ReRank, running `SELECT * FROM provider_models` on the Dify DB returns the following result, indicating that it is configured correctly.
```
... | ...  | localai       | <<model_name>> | reranking       | {"server_url": "http://xxx.xxx.xxx.xxx/v1"} | t        | 2024-05-17 17:10:23 | 2024-05-23 07:26:34.66699
```
However, it does not show up in the model provider settings and cannot be used as a model.

It seems like the API endpoint `console/api/workspaces/current/models/model-types/rerank` is not functioning with the registered LocalAI Reranker.

Request: `http://xxx/console/api/workspaces/current/models/model-types/rerank`
Response:
```
{data: [{provider: "nvidia", label: {zh_Hans: "API Catalog", en_US: "API Catalog"},…},…]}
0: {provider: "nvidia", label: {zh_Hans: "API Catalog", en_US: "API Catalog"},…}
1: {provider: "cohere", label: {zh_Hans: "Cohere", en_US: "Cohere"},…}
2: {provider: "jina", label: {zh_Hans: "Jina", en_US: "Jina"},…}
```

**Environment:**
- Dify version: 0.6.8
- LocalAI version: 2.15.0

**Comment by Sarmingsteiner** [comment]
@edisonzf2020 : Could you please point me out how you managed to install the reranking model `mxbai-rerank-large-v1` in LocalAI?

**Comment by gakugaku** [comment]
@Sarmingsteiner 
This issue has been resolved in #4652, and it should work correctly in the latest version.

Here is how I managed to install a reranking model in LocalAI. Example for Docker and Japanese Reranker model.

### LocalAI Configuration

`compose.yaml`
```yaml
services:
  local-ai:
    image: localai/localai:v2.16.0
    environment:
      MODELS_PATH: /models
    volumes:
      - ./data/LocalAI/models:/models:cached
```

`./data/LocalAI/models/reranker.yaml`
You can find more details in the [📈 Reranker | LocalAI documentation](https://localai.io/features/reranker/).

```yaml
name: japanese-reranker
backend: rerankers
parameters:
  model: hotchpotch/japanese-reranker-cross-encoder-small-v1
```

When you start and run this configuration, the model will be downloaded automatically.

### Dify Configuration

Select the LocalAI provider and configure it as follows:

- **Model Type:** Rerank
- **Model Name:** In the example above, it would be `japanese-reranker`
- **Server URL:** `http://<your-server>/v1` (make sure to include `/v1`)
